src/accelerate/commands/launch.py

```diff
@@ -877,15 +877,15 @@ def launch_command(args):
         if not hasattr(args, "use_cpu"):
             args.use_cpu = args.cpu
 
-    if args.num_cpu_threads_per_process is None:
+    if args.num_cpu_threads_per_process == 1 and args.use_cpu:
```

Comment:
This means a chosen value of 1 is erased. Let's leave the default at `None` and change it at 1 when not in distributed CPU, or the other default when in distributed CPU.

Reply:
Adjusted :)